data_project/README.md

```diff
@@ -28,7 +28,7 @@ no longer needed, which supports CI/CD best practices when working with Snowflak
     * what files should be included in the version creation process. In the `include_definitions` list you can:
       * include a specific file by passing its relative path to the root of the project, e.g. `definitions/account_objects.sql`
       * include all files from a specific directory, e.g. `definitions/.*`
-    * values for template variables. Definition files (templates) are rendered as Jinja2 variables during the project execution process. Variable values are defined in the `template_variables` list, e.g. `example_db_name: "db1"`
+    * list of configurations that allow users to use templating in definition files. Definition files (templates) are rendered as Jinja2 variables during the project execution process. Each configuration contains a set of key-value pairs, e.g. `example_db_name: "db1"`
```

Comment:
Maybe it's better to link to [the docs](https://docs.snowflake.com/LIMITEDACCESS/data-projects/manage-data-projects#project-manifest-file) instead of repeating the same information. The description may misalign.